rust/routee-compass/src/plugin/input/default/load_balancer/weight_heuristic.rs

```diff
@@ -0,0 +1,38 @@
+use crate::plugin::{input::input_json_extensions::InputJsonExtensions, plugin_error::PluginError};
+use routee_compass_core::util::{
+    geo::haversine,
+    unit::{as_f64::AsF64, Distance, DistanceUnit},
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+#[serde(rename_all = "snake_case", tag = "type")]
+pub enum WeightHeuristic {
+    /// computes a weight directly as the haversine distance estimation between
+    /// trip origin and destination, in meters.
```

Comment:
I think this should be kilometers right?

Reply:
oops, yes, originally was meters. good catch.